docs/database/database-metrics.md

```diff
@@ -106,6 +106,8 @@ of `[ 0.001, 0.005, 0.01, 0.05, 0.1, 0.5, 1, 5, 10 ]`.
 | [`db.operation.name`](/docs/attributes-registry/db.md) | string | The name of the operation or command being executed. [5] | `findAndModify`; `HMSET`; `SELECT` | `Conditionally Required` [6] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`error.type`](/docs/attributes-registry/error.md) | string | Describes a class of error the operation ended with. [7] | `timeout`; `java.net.UnknownHostException`; `server_certificate_invalid`; `500` | `Conditionally Required` If and only if the operation failed. | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 | [`server.port`](/docs/attributes-registry/server.md) | int | Server port number. [8] | `80`; `8080`; `443` | `Conditionally Required` [9] | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| [`thread.id`](/docs/attributes-registry/thread.md) | int | Current "managed" thread ID (as opposed to OS thread ID). | `42` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`user.id`](/docs/attributes-registry/user.md) | string | Unique identifier of the user. | `S-1-5-21-202424912787-2692429404-2351956786-1000` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
the database user is under discussion, please check  https://github.com/open-telemetry/semantic-conventions/issues/1142

Reply:
Removed the `user.id` attribute for now

Reply:
@lmolkova Sorry to bother you, but any further feedback on this PR? And would you be able to reopen it?